### PR TITLE
adding plist running as daemon

### PR DESCRIPTION
**Pull request by mulbane**
adding plist entry for cjdns formula for persistence

### CODE REVIEW

File: Library/Formula/cjdns.rb [hunk 1]
```diff
@@ -19,6 +19,42 @@ def install
     (share+"test").install "build_darwin/test_testcjdroute_c" => "cjdroute_test"
   end
 
+  def caveats
+    s = <<-EOS.undent
+      !!!IMPORTANT!!!
+      Before using cjdns, you must generate a configuration file and find peers manually.
+      Once you have found eligible peers, use the following command to generate a config that is appropriate for Launchd
+        sudo bash -c "cjdroute --genconf | sed 's@\\"noBackground\\":0@\\"noBackground\\":1@' > /etc/cjdroute.conf"
```
Comment (mikemcquaid) [inline Library/Formula/cjdns.rb]: Can this not be run automatically? Also, can the configuration file not use `HOMEBREW_PREFIX/etc`?

File: Library/Formula/cjdns.rb [hunk 2]
```diff
@@ -19,6 +19,42 @@ def install
...
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>sudo</string>
+          <string>bash</string>
+          <string>-c</string>
+          <string>/usr/local/bin/cjdroute &lt; /etc/cjdroute.conf</string>
```
Comment (mikemcquaid) [inline Library/Formula/cjdns.rb]: You should use `opt_bin` rather than hardcoding `/usr/local` here.

File: Library/Formula/cjdns.rb [hunk 3]
```diff
@@ -19,6 +19,42 @@ def install
...
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>sudo</string>
```
Comment (mikemcquaid) [inline Library/Formula/cjdns.rb]: You can't rely on this command running without `sudo`. It should be a startup plist if it needs root.

**Comment by DomT4** [comment]
Closing as dead. We'll happily review a new PR with the various comments addressed here, Thanks for the submission!

**Pull request closed without merging**